stable/prometheus-cloudwatch-exporter/templates/clusterrole.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.rbac.create }}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
If we want to support multiple API version we could also do it like:

```
{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
apiVersion: rbac.authorization.k8s.io/v1
{{- else }}
apiVersion: rbac.authorization.k8s.io/v1beta1
{{- end }}
```

Reply:
@velothump Could you implement it like suggested and fix the conflicts? I'll do my best to review it faster.